### PR TITLE
Update contact form

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated contact form on `/kubernetes`, `/kubernetes/what-is-kubernetes`, `/kubernetes/compare`, `/kubernetes/install`, and `/kubernetes/managed`
- copydoc: https://docs.google.com/document/d/1u6IkVfDNNwdz4XgHU6FSCnUFfVtfTMNMmtINIPT31cU/edit#

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10368.demos.haus/kubernetes
- Visit all relevant pages (see above) and check confirm the titles match the copydoc
- Try to submit without filling out filling all relevant fields, check that it correctly fails
- Do the same but with all fields filled and submit 

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4429

## Screenshots

[If relevant, please include a screenshot.]

### CODE REVIEW

File: templates/shared/forms/interactive/kubernetes.html [hunk 1]
```diff
@@ -2,215 +2,17 @@
...
-        <a class="pagination__link--previous p-button--neutral" href="">Previous</a>
-        <a class="pagination__link--next p-button--positive" href="">Next</a>
-      </div>
-    </div>
-
-    <div class="js-pagination js-pagination--3 u-hide">
       <div class="u-sv3 js-formfield">
-        <h3 class="p-heading--five">Have we missed something?</h3>
-        <textarea id="particular-sectors" aria-label="Have we missed something?" rows="3" placeholder="Are there particular sectors which are important to you such as industrial, automotive, display, retail, home, office, data center, networking, robotics?"></textarea>
+        <h3 class="p-heading--five">Let us know about your Kubernetes use case</h3>
```
Comment (steverydz) [inline templates/shared/forms/interactive/kubernetes.html:9]: According to the copy doc this should be:

> Tell us more about your Kubernetes use case

File: templates/shared/forms/interactive/kubernetes.html [hunk 2]
```diff
@@ -2,215 +2,17 @@
...
+        <textarea id="particular-sectors" aria-label="Let us know about your Kubernetes use case" rows="3" placeholder="Anything you’d like to communicate about your needs or interests?"></textarea>
```
Comment (steverydz) [inline templates/shared/forms/interactive/kubernetes.html:10]: We usually encode ’ e.g. \&rsquo;

File: templates/shared/forms/interactive/kubernetes.html [hunk 3]
```diff
@@ -259,15 +61,14 @@ <h3>Your comments</h3>
             </div>
 
             <div class="pagination">
-              <a class="pagination__link--previous p-button--neutral" href="">Previous</a>
               <button type="submit" class="pagination__link--next p-button--positive mktoButton" aria-label="Submit">Let's discuss</button>
```
Comment (steverydz) [inline templates/shared/forms/interactive/kubernetes.html:64]: Please encode apostrophe e.g. \&rsquo;

File: templates/shared/forms/interactive/kubernetes.html [hunk 4]
```diff
@@ -276,19 +77,19 @@ <h3 class="p-heading--two">Thank you for enquiring about Kubernetes</h3>
...
             {{
-              image(
-              url="https://assets.ubuntu.com/v1/cbae9a60-thank_you_orange_cmyk.svg",
-              alt="smile",
-              width="200",
-              height="200",
-              hi_def=True,
-              loading="auto",
-              ) | safe
+            image(
```
Comment (steverydz) [inline templates/shared/forms/interactive/kubernetes.html:80]: Why remove the indent?

**Review by steverydz — COMMENTED** [review]
Looks good. Just a few small changes in the code.

**Comment by pmahnke** [comment]
On ubuntu.com, we are using a curly quote script, so you don't have to
encode '''s

File: templates/shared/forms/interactive/managed-kubernetes.html [hunk 5]
```diff
@@ -151,7 +34,7 @@ <h3 class="p-heading--five">How should we get in touch?</h3>
                 <input class="mktoField" value="yes" id="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox" />
                 <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</label>
               </li>
-              <li class="p-list__item">In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</li>
+              <li class="p-list__item">In submitting this form, I confirm that I have read and agree to Canonical's<a href="/legal/terms-and-policies/privacy-policy"> Privacy Policy</a></li>
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/managed-kubernetes.html:37]: Probably should be a space before the link here

File: templates/shared/forms/interactive/kubernetes.html (hunk 4 above)
Comment (mtruj013) [reply templates/shared/forms/interactive/kubernetes.html]: It wasn't intentional, but it's fixed now

**Review by steverydz — APPROVED** [review]
LGTM 👍

File: templates/shared/forms/interactive/managed-kubernetes.html (hunk 5 above)
Comment (mtruj013) [reply templates/shared/forms/interactive/managed-kubernetes.html]: Fixed👍🏻

**Pull request merged**